oracle/src/oracleimporter.cpp

```diff
@@ -33,13 +33,10 @@ bool OracleImporter::readSetsFromByteArray(const QByteArray &data)
 
     while (it.hasNext()) {
         map = it.next().toMap();
-        edition = map.value("code").toString();
+        edition = map.value("code").toString().toUpper();
         editionLong = map.value("name").toString();
         editionCards = map.value("cards");
-        setType = map.value("type").toString();
-        // capitalize set type
-        if (setType.length() > 0)
-            setType[0] = setType[0].toUpper();
+        setType = map.value("type").toString().toUpper();
```

Comment:
`Set type` looks weird now:
![set type](https://user-images.githubusercontent.com/9874850/49134569-4b9f5a80-f2e4-11e8-8baf-b840b664e2f9.png)

Reply:
Fixed, thank you